charts/rancher-istio/1.1.5-rancher1/charts/certmanager/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Certmanager uses ACME to sign certificates. Since Istio gateways are
+# mounting the TLS secrets the Certificate CRDs must be created in the
+# istio-system namespace. Once the certificate has been created, the
+# gateway must be updated by adding 'secretVolumes'. After the gateway
+# restart, DestinationRules can be created using the ACME-signed certificates.
+enabled: false
+image:
+  repository: quay.io/jetstack/cert-manager-controller
```

Comment:
I think it's better if we can push the image to the rancher hub, otherwise, the current certmanager image config will not work in the air-gap environment.